lib/core/src/Cardano/Wallet.hs

```diff
@@ -309,6 +319,10 @@ data ErrEstimateTxFee e
     | ErrEstimateTxFeeCoinSelection (ErrCoinSelection e)
     deriving (Show, Eq)
 
+newtype ErrListUTxOStatistics
+    = ErrListUTxOStatisticsNoSuchWallet ErrNoSuchWallet
+    deriving (Show, Eq)
```

Comment:
Minor note: here the wrapper is mostly unecessary, we could have simply returned `ErrNoSuchWallet`.
I think either approach have good arguments, using already a `ErrListUTxOStatistics` is quite consistent with naming and makes it easy to add more branches later on.

Reply:
I am advocate of latter argument. Stick to already established naming convention